.github/workflows/functions-v2-ocr.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: functions-v2-ocr
```

Comment:
We should also add this to the list of [workflows](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/workflows/utils/workflows.json), for tracking and so it will be regenerated as part of bulk updates to the workflows.

Also, so that it can be regenerated in the future without any manual changes (e.g. line 40), I would consider making this workflow rooted at functions/v2/ocr/app. So, the entry in `workflows.json` would have that path, and the generate command would look like:
`node .github/workflows/utils/generate.js -s functions/v2/ocr/app`.

Reply:
Updated. Thanks!